javascript/javascript1/week4/palindromeSubstring.js

```diff
@@ -0,0 +1,37 @@
+//A palindrome is a word, number, phrase, or other sequence of characters which reads the same backward as forward. eg: madam
+let longestPalindrome = function(string) {
```

Comment:
to be fair I don't understand the solution... Have you taught of a simple approach first...In Computer Science you can start solving a problem by a brute force approach...meaning in this case that we first generate all the substrings of a given string and then check each one of them if it's a palidrom and if that is the case we remember the size of it
This is a pseudocode bellow


**1. for (i = 0; i < string.length; i++) {
2.       for (j=i+1; j < string.length; j++) {
3.             check if string[i,j] is palindrom
4.            if it's true then check if the size of string[i,j] is higher then the
5.           current value and store it.
6.     }
7. }**